tembo-operator/src/apis/coredb_types.rs

```diff
@@ -89,6 +89,84 @@ pub struct ServiceAccountTemplate {
     pub metadata: Option<ObjectMeta>,
 }
 
+/// AzureCredentials is the type for the credentials to be used to upload files to Azure Blob Storage.
+#[derive(Serialize, Deserialize, Clone, Debug, Default, JsonSchema)]
+pub struct AzureCredentials {
```

Comment:
are these types already present in the cnpg file, or could be kopium imported from the CNPG crd

Reply:
Yes they are already present. The S3Credentials  values (right below this section) are also present there, but it looks like we copied them in and renamed slightly. I followed the same pattern here. Happy to go with either